_data/profile_versions.yaml

```diff
@@ -20,7 +20,7 @@ ComputationalTool:
   name: ComputationalTool
   status: active
 ComputationalWorkflow:
-  latest_publication: null
+  latest_publication: 2.0-DRAFT
```

Comment:
@sahar-frikha how is the new DRAFT version determined? I was expecting to see 1.1-DRAFT

Reply:
@ljgarcia -- That part was automated based on the name of the JSON-LD file added to the specifications repository. The JSON-LD file was generated by @nsjuty during the biohackathon and was likely given the 2.0 name based on the remaining GitHub issues he was addressing for the [project](https://github.com/BioSchemas/specifications/projects/11). I added the term `DRAFT` to the file name as the version had not gone through the formal review process.

Reply:
To my knowledge, the ComputationalWorkflow has some problems with the automated rendering, which is why no one was tagged to review it. Please do NOT merge this pull request.

Reply:
@gtsueng I will not merge this one and will remove the other three PR about ComputationalWorkflow (which have the same issue with the version)
PRs to be closed with no further action:
* https://github.com/BioSchemas/bioschemas.github.io/pull/710
* https://github.com/BioSchemas/bioschemas.github.io/pull/709
* https://github.com/BioSchemas/bioschemas.github.io/pull/708
This was agreed with @sahar-frikha during a meeting to sort out issues with the automated PRs, in particular regarding how the version was updated